app/portainer/home/EnvironmentList/KubeconfigButton.tsx

```diff
@@ -20,9 +34,42 @@ export function KubeconfigButton({ environments }: Props) {
   }
 
   return (
-    <Button onClick={handleClick}>
-      <i className="fas fa-download space-right" /> kubeconfig
-    </Button>
+    <div>
+      <Button onClick={handleClick}>
+        <i className="fas fa-download space-right" /> kubeconfig
+      </Button>
+      <DialogOverlay
+        className="center-dialog"
+        isOpen={showDialog}
+        aria-label="Kubeconfig View"
+        role="dialog"
+      >
+        <div className="modal-content">
+          <div className="modal-header">
+            <button
+              type="button"
+              className="bootbox-close-button close"
+              aria-hidden="true"
```

Comment:
I think this should be displayed always

Reply:
Yeah, Just googled what is aria-hidden. I have removed this line.